src/languages/es.ts

```diff
@@ -1012,6 +1012,25 @@ export default {
         },
         cardDetailsLoadingFailure: 'Se ha producido un error al cargar los datos de la tarjeta. Comprueba tu conexión a Internet e inténtalo de nuevo.',
     },
+    workflowsPage: {
+        workflowTitle: 'Gasto',
+        workflowDescription: 'Configure un flujo de trabajo desde el momento en que se produce el gasto, incluida la aprobación y el pago.',
+        delaySubmissionTitle: 'Retrasar las presentaciones',
+        delaySubmissionDescription: 'Los gastos se comparten de inmediato para una mejor visibilidad del gasto. Establece una cadencia más lenta si es necesario.',
+        addApprovalsTitle: 'Añadir aprobacións',
+        approver: 'Aprobador',
+        submissionFrequency: 'Frecuencia de envío',
+        weeklyFrequency: 'Semanal',
+        monthlyFrequency: 'Mensual',
+        twiceAMonthFrequency: 'Dos veces al mes',
+        byTripFrequency: 'Por viaje',
+        manuallyFrequency: 'Manualmente',
+        dailyFrequency: 'Diario',
+        connectBankAccount: 'Conectar cuenta bancaria',
+        addApprovalsDescription: 'Requiere una aprobación adicional antes de autorizar un pago.',
+        makeOrTrackPaymentsTitle: 'Realizar o seguir pagos',
+        makeOrTrackPaymentsDescription: 'Añada un pagador autorizado para los pagos realizados en Expensify, o simplemente realice un seguimiento de los pagos realizados en otro lugar.',
+    },
```

Comment:
No translations have been provided. I've put the keys in the spanish translations file because we need to.

cc @luacmartins do you have more info on that or these translations look good to you?

Reply:
We need to get these reviewed. Some of these look a bit odd to me haha. Posted [here](https://expensify.slack.com/archives/C21FRDWCV/p1708352018471839)!

Reply:
I have no Spanish skills at all, so they _might_ be not that good 😄

Reply:
Okay I've fixed the translations thanks to Ionatan's help :)